persistence/soft-index/src/test/java/org/infinispan/persistence/sifs/SoftIndexFileStoreTest.java

```diff
@@ -106,6 +113,55 @@ public void testStopStartAndMultipleWrites() {
       assertEquals("v2", store.load("k1").getValue());
    }
 
+   // test for ISPN-5743
+   public void testStopStartWithRemoves() {
```

Comment:
In the stack trace, there's a missing thread named BCS-Compactor; is there any log with exception?